app/views/users/mfa_selection/index.html.erb

```diff
@@ -4,6 +4,21 @@
 
 <p class="maxw-mobile-lg margin-bottom-0"><%= @presenter.intro %></p>
 
+<h2 class="margin-top-2 margin-bottom-1">
+  <%= t('headings.account.two_factor') %>
+</h2>
+
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    <ul class="usa-icon-list">
+      <% @presenter.options.each do |option| %>
+        <% if option.mfa_configuration_count > 0 %>
+          <%= render partial: 'partials/multi_factor_authentication/selected_mfa_option', locals: { option: option } %>
+        <% end %>
+      <% end %>
+    </ul>
+  </div>
```

Comment:
On second thought, do we actually need the grid container here, if it's always shown as full-width?

```suggestion
<ul class="usa-icon-list">
  <% @presenter.options.each do |option| %>
    <% if option.mfa_configuration_count > 0 %>
      <%= render partial: 'partials/multi_factor_authentication/selected_mfa_option', locals: { option: option } %>
    <% end %>
  <% end %>
</ul>
```